trunk/src/app/srs_app_http_conn.hpp

```diff
@@ -136,8 +136,18 @@ class SrsHttpxConn : public ISrsConnection, public ISrsStartable, public ISrsHtt
     SrsHttpConn* conn;
     // We should never enable the stat, unless HTTP stream connection requires.
     bool enable_stat_;
-public:
-    SrsHttpxConn(bool https, ISrsResourceManager* cm, ISrsProtocolReadWriter* io, ISrsHttpServeMux* m, std::string cip, int port);
+    // ssl key & cert file
+    const std::string ssl_key_file;
```

Comment:
For member field, should end width `_`, so should rename to `ssl_key_file_` and `ssl_cert_file_`

Reply:
fixed.